packages/manager/src/features/DataStream/Shared/FormSubmitBar/FormSubmitBar.tsx

```diff
@@ -0,0 +1,84 @@
+import { Box, Button, Divider, Paper, Stack, Typography } from '@linode/ui';
+import { capitalize } from '@linode/utilities';
+import * as React from 'react';
+
+import { getDestinationTypeOption } from 'src/features/DataStream/dataStreamUtils';
+import { StyledHeader } from 'src/features/DataStream/Shared/FormSubmitBar/FormSubmitBar.styles';
+
+import type { DestinationType } from '@linode/api-v4';
+import type { FormMode, FormType } from 'src/features/DataStream/Shared/types';
+
+type StreamFormSubmitBarProps = {
```

Comment:
We tend to prefer using interface throughout the codebase

```suggestion
interface StreamFormSubmitBarProps {
```

Reply:
Ok, I'll change it here to interface and refactor existing types to interfaces in my next ticket where I'm doing a refactor